website/docs/cloud-docs/integrations/run-tasks/index.mdx

```diff
@@ -157,6 +157,12 @@ To get started, [create a free Snyk account](https://snyk.io/) and follow the in
 
 Sophos Factory works with Terraform Cloud run tasks to enable IaC security tools including OPA, BridgeCrew’s Checkov, Accurics, TerraScan through a public solution catalog of prebuilt DevSecOps pipelines. To get started, follow the [Integrate with Terraform Cloud](https://docs.refactr.it/docs/tutorials/terraform-cloud/) tutorial in the Sophos documentation.
 
+### SpectralOps
+
+[SpectralOps](https://spectralops.io/) SpectralOps integration with Terraform Cloud & Terraform Enterprise enables security teams to empower their engineers to monitor, classify, and protect their code & infrastructure for security misconfiguration.
+
+To get started, follow the instructions in the [Terraform Cloud integration](https://get.spectralops.io/docs/integration/tfc/) user documentation.
+
 ### Styra
 
 Styra DAS works with Terraform Cloud to ensure that plans for an apply action are always evaluated against your DAS Terraform OPA policies. To get started, refer to the [Terraform Cloud](https://docs.styra.com/tutorials/terraform-cloud/introduction) content in the Styra documentation.
```

Comment:
```suggestion
Styra DAS works with Terraform Cloud to always evaluate plans for an apply action against your DAS Terraform OPA policies. To get started, refer to the [Terraform Cloud](https://docs.styra.com/tutorials/terraform-cloud/introduction) content in the Styra documentation.
```

Suggest changing "to ensure that plans for an apply action are always evaluated" to "to always evaluate" to avoid a passive voice style issue.